src/app/static/src/js/components/permissions/roleList.vue

```diff
@@ -65,54 +51,22 @@
             availableUsersForRole: function(role) {
                 return this.availableUsers.filter(u => role.members.map(m => m.email).indexOf(u) < 0)
             },
-            canRemoveOrDeleteRole: function(role) {
-                return this.canRemoveRoles || (this.canDeleteRoles && role !== "Admin")
+            canRemoveRole: function(role) {
```

Comment:
So this means user also won't be able to remove report permissions from Admin role - that's probably ok although Admin role shouldn't need scope report permission as should have global.